src/js/profile/facilities/facility_controller.js

```diff
@@ -112,13 +131,39 @@ export class FacilityController extends Component {
         })
     }
 
+    prepareFailMessage() {
+        if ($('.rich-data-content .location__facilities .location__facilities_error').length > 0) {
+            return;
+        }
+
+        let failMsg = document.createElement('div');
+        $(failMsg).addClass('location__facilities_error').addClass('hidden');
+        $(failMsg).html('Failed to load this data');
+
+        $(failMsg).css('display', 'block');
+        $(failMsg).css('color', '#666');
+        $(failMsg).css('width', '100%');
+        $(failMsg).css('background-color', '#f0f0f0');
+        $(failMsg).css('padding', '4px 6px');
+        $(failMsg).css('font-size', '0.9em');
+        $(failMsg).css('border-radius', '2px');
+        $(failMsg).css('margin-right', '6px');
+        $(failMsg).css('margin-left', '6px');
+
+        $('.rich-data-content .location__facilities').append(failMsg);
+    }
+
     getAndAddFacilities(activeVersion) {
         if (this.model.api !== null) {
             this.model.api.getThemesCount(this.model.profileId, this.model.geography.code, activeVersion.model.name)
                 .then((data) => {
                     this.model.themes = data;
                     this.addFacilities();
                 })
+                .catch(() => {
+                    this.isLoading = false;
+                    this.isFailed = true;
+                })
```

Comment:
Please log whatever you catch here in sentry - especially since this is catching more than just network issues, right? this is catching everything.

Reply:
done